src/pylimer_tools/io/bead_spring_parameter_provider.py

```diff
@@ -131,6 +133,14 @@ def get_bead_density(self) -> float:
             # assert dimensionless density
             assert density.check("[dimensionless]")
             return density.magnitude
+    
+    def get_characteristic_ratio(self) -> float:
+        """
+        Returns the characteristic ratio of the polymer.
+        """
+        characteristic_ratio = self.get("C_inf")
+        assert characteristic_ratio.check("")
+        return characteristic_ratio.magnitude
```

Comment:
Other values (e.g. the mean bead distance "\<b\>") don't have their own getter method, only if they are computed from others, which is not the case for the characteristic ratio. I admit, this is not a particularly clean interface, which is why I have now implemented a more consistent getter using `__getattr__`.